packages/smart-forms-renderer/src/stories/itemTypes/Attachment.stories.tsx

```diff
@@ -17,7 +17,11 @@
 
 import type { Meta, StoryObj } from '@storybook/react-vite';
 import BuildFormWrapperForStorybook from '../storybookWrappers/BuildFormWrapperForStorybook';
-import { qAttachmentBasic } from '../assets/questionnaires';
```

Comment:
also delete all unused files like ../assets/questionnaires if they are not used anywhere